### DISCUSSION TITLE
I need help "-"

### DISCUSSION THREAD

**Discussion by sebastianjnuwu** [npm/feedback#726] [CLI] [answered]
It was just another normal day where I would install my dependencies and develop some things but I'm not getting it, yes I'm not ;-; 

![](https://media.discordapp.net/attachments/991506128972304414/996366672317386772/Screenshot_20220712-074416.jpg)

I don't have root (sudo) on my phone

**Comment by MylesBorins** [comment, marked as answer]
Hey @sebastianjnuwu we don't offer general support here. This discussion board is about suggesting features and discussing improvements to npm.

Stackoverflow or GitHub.community will be better places to get help with this. It looks like npm cache is pointing to a folder which is root... [you may want to configure you cache directory](https://docs.npmjs.com/cli/v8/using-npm/config#cache)
